src/browser-lib/getBrowserConfig.ts

```diff
@@ -147,7 +147,7 @@ const envVars = satisfies<Record<string, EnvVar>>()({
     default: null,
   },
   searchApiUrl2023: {
-    value: process.env.NEXT_PUBLIC_SEARCH_API_URL_2023,
+    value: `${process.env.NEXT_PUBLIC_SEARCH_API_URL_2023}/v2`,
```

Comment:
wondering if this should be a part of the env var instead of added in the code?

Reply:
Yeah i originally had the change in config, this was @mantagen suggestion. Although he suggested I could reach out for other opinions. Happy to move back to config if that's better.

Reply:
we discussed this and think it makes sense to handle in the code, so any version cahnges can be properly tested in a PR